libs/designer/src/lib/ui/templates/templateoverview.tsx

```diff
@@ -0,0 +1,229 @@
+import { useIntl } from 'react-intl';
+import { useDispatch, useSelector } from 'react-redux';
+import type { AppDispatch, RootState } from '../../core/state/templates/store';
+import type { Template } from '@microsoft/logic-apps-shared';
+import { getPropertyValue, unmap } from '@microsoft/logic-apps-shared';
+import { Text } from '@fluentui/react-components';
+import { QuickViewPanelHeader } from '../panel/templatePanel/quickViewPanel/quickViewPanel';
+import { ConnectionsList } from './connections/connections';
+import { useFunctionalState } from '@react-hookz/web';
+import type { WorkflowTemplateData } from '../../core/actions/bjsworkflow/templates';
+import { openQuickViewPanelView } from '../../core/state/templates/panelSlice';
+import { TemplatePanel } from '../panel/templatePanel/templatePanel';
+import { TemplatesPanelFooter } from '@microsoft/designer-ui';
+import { workflowTab } from '../panel/templatePanel/quickViewPanel/tabs/workflowTab';
+
+export const TemplateOverview = ({ createWorkflow }: { createWorkflow: CreateWorkflowHandler }) => {
+  useEffect(() => setLayerHostSelector('#msla-layer-host'), []);
+  const intl = useIntl();
+  const dispatch = useDispatch<AppDispatch>();
+  const [workflowDetails, setWorkflowDetails] = useState<string | undefined>(undefined);
+  const [showCreatePanel, setShowCreatePanel] = useState<boolean>(false);
+  const { templateName, workflowAppName, manifest, connections, workflows } = useSelector((state: RootState) => ({
+    templateName: state.template.templateName,
+    workflowAppName: state.workflow.workflowAppName,
+    manifest: state.template.manifest,
+    connections: state.template.connections,
+    workflows: state.template.workflows,
+  }));
+  const { title, description, details, detailsDescription } = manifest as Template.Manifest;
+  const resources = {
+    by: intl.formatMessage({
+      defaultMessage: 'By',
+      id: '+5Jp42',
+      description: 'Title for publisher',
+    }),
+    type: intl.formatMessage({
+      defaultMessage: 'Type',
+      id: 'k/X2ml',
+      description: 'Title for solution type',
+    }),
+  };
+  const info = {
+    [resources.by]: getPropertyValue(details, 'by'),
+    [resources.type]: getPropertyValue(details, 'type'),
+  };
+  const templateHasConnections = Object.keys(connections).length > 0;
+
+  const showDetails = (workflowId: string) => {
+    dispatch(openQuickViewPanelView());
+    setWorkflowDetails(workflowId);
+  };
+
+  // TODO: Need to open new create panel for multi workflow here.
+  const footerContentProps = workflowTab(intl, dispatch, true, () => setShowCreatePanel(true), {
+    templateId: templateName ?? '',
+    workflowAppName,
+  }).footerContent;
+  return (
+    <>
+      <QuickViewPanelHeader title={title} description={description} details={info} features={detailsDescription} />
+      <div className="msla-template-overview" style={{ marginTop: '-34px' }}>
```

Comment:
Nit: can we put the `margin-top` style inside the class?

Reply:
Nit: Would this be the best way to handle the styling? Negative marginTop and marginBottom seem to be not the best unless absolutely necessary. There should be better ways to handle this, for instance, if the marign is coming from QuickVIewPanelHeader, we should remove margin from the QuickViewPanelHeader.

You can look at CreateWorkflowPanel for reference. This removes default margin given by the react markdown component.

![image](https://github.com/user-attachments/assets/61e79538-b462-404c-bba2-3269c7b3dc95)

Otherwise, once merged, I can come back to this css afterwards as well.

Reply:
I think I need to discuss this, on how we can share the single workflow components here, specifically the common tabs in panel... I can provide explicit css in header component that way we do not have to do it...
I am going to file an item and track the css changes through it, for now will add a TODO in code so we can revisit when functionality is done.